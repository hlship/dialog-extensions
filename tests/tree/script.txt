talk to tree
 animal
vertebrates
mammals
convinfo
change subject
primates
minerals
plants
convinfo
%% Because all the top-level quips (animal, plants, minerals)
%% have been exhausted, there's no new thread to switch to.
change the subject