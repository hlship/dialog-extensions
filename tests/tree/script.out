

Altar of the Tree of Knowledge
You have reached the altar of the Tree of Knowlege. The tree's branches rise up,
out of the deep ravine, towards the light; its roots have worked their way into
the living rock, and entagle the great stone altar.

> talk to tree
The voice of the tree eminates from all around. "You may ask about things
animal, vegetable, or mineral."

You could ask about animals, ask about plants, or ask about minerals.

>  animal
You ask about animals.

Are you interested in vertebrates or invertebrates?

You could ask about plants, ask about minerals, ask about vertebrates, or ask
about invertebrates.

> vertebrates
What are vertebrates?

Animals with backbones are vertebrates, such as mammals, fish, and lizards.

You could ask about invertebrates, ask about mammals, or ask about fish.

> mammals
What are mammals?

Mammals are warm blooded, bear their young alive, and produce milk. Primates
like you are mammals.

You could ask about primates or ask about fish.

> convinfo
Conversation partner: #tree -- the tree of knowledge
Quips:
  Current: #mammal
  Previous: #vertebrate
  Grandparent: #animal

Discussable quips:
  #vegetable (changes the subject)
  #mineral (changes the subject)
    Followers: #metal, #stone
  #invertebrate
    Follows: #animal
  #primate (relevant)
    Follows: #mammal
  #fish (relevant)
    Follows: #vertebrate

> change subject
You could ask about plants or ask about minerals.

> primates
What's a primate?

Larger brains, opposable thumbs. An interesting experiment you could say.

A moment passes.

You are a primate. Rather less well adapted than a chimp or a great ape, I
believe.

You could ask about plants, ask about minerals, ask about invertebrates, or ask
about fish.

> minerals
What are minerals?

Minerals are non-living elements, many of which are building blocks of life.
Metals, stone, and crystals are types of minerals.

You could ask about plants, ask about invertebrates, ask about fish, ask about
metals, or ask about stone.

> plants
Plants and vegetables?

Mosses, land plants, and aquatic plants.

You could ask about invertebrates, ask about fish, ask about metals, or ask
about stone.

> convinfo
Conversation partner: #tree -- the tree of knowledge
Quips:
  Current: #vegetable
  Previous: #mineral
  Grandparent: #vegetable

Discussable quips:
  #invertebrate
    Follows: #animal
  #fish
    Follows: #vertebrate
  #metal (relevant)
    Follows: #mineral
  #stone (relevant)
    Follows: #mineral

> change the subject
You can think of no changes of subject now.

> 
